### ISSUE TITLE
[Question] VK Music Is Out

### ISSUE THREAD

**Issue by ronaiza-cardoso** [amirfrsd/Gramaphone#1]
VK Music is out on brazil, do you know some other service?

**Comment by amirfrsd** [comment]
Hey, Yeah i know, I'm working on Spotify, We could bring back the VK Music to Brazil in months or so ...

**Comment by ronaiza-cardoso** [comment]
Are you responsible for the site?

**Comment by amirfrsd** [comment]
Nope, But you can run our VK Music Crawler on your servers, Soon... 👍
